frontend/src/components/social/logsPreview/LogPreviewBox.jsx

```diff
@@ -115,7 +118,9 @@ const Box = styled.div`
             0 0 0 0.3em ${(props) => props.$bgColor};
     }
 
-    ${ifMobile} {
+    ${ifMobile}${ifTablet} {
```

Comment:
```suggestion
    ${ifTablet} {
```

`ifTablet`로 스타일을 지정하면 그 이하의 화면 크기에도 스타일이 적용됩니다.